src/classes/BGE_ConfigurationWizard_CTRL.cls

```diff
@@ -191,38 +191,40 @@ public with sharing class BGE_ConfigurationWizard_CTRL {
             'EMAIL',
             'MULTIPICKLIST',
             'PHONE',
-            'URL'
+            'URL',
+            'REFERENCE'
```

Comment:
This is a nice-to-have but would it be good to switch these out for the Schema.DisplayType enum values (or whatever the appropriate source is?)

Reply:
Yes, true. Good refactor.